### PR TITLE
ci: run tests for pull requests as well

### PR DESCRIPTION
**Pull request by wolfy1339**
This should unblock #379

### CODE REVIEW

**Comment by thomas-maqua-db** [comment]
@wolfy1339 @oscard0m Sorry to bother you, but is there a way to speed this up? It would be really helpful if this and https://github.com/octokit/action.js/pull/379 could get merged soon.

**Comment by wolfy1339** [comment]
I can't personally do much, as I need a review from another maintainer in order to merge this

**Review by oscard0m — APPROVED** [review]

**Comment by oscard0m** [comment]
> @wolfy1339 @oscard0m Sorry to bother you, but is there a way to speed this up? It would be really helpful if this and #379 could get merged soon.

Sorry for the delay in this review. Merged!

**Pull request merged**
